allennlp/commands/_checklist_internal.py

```diff
@@ -0,0 +1,204 @@
+"""
+The `checklist` subcommand allows you to conduct behavioural
+testing for your model's predictions using a trained model and its
+[`Predictor`](../predictors/predictor.md#predictor) wrapper.
+"""
```

Comment:
This is a case where git does a better job than the github UI; in particular, this is a 100% copy of the file as it existed before ec96bef1:
```
[cf ec96bef1] create dummy command if checklist not available
 4 files changed, 51 insertions(+), 236 deletions(-)
 copy allennlp/commands/{checklist.py => _checklist_internal.py} (100%)
 rewrite allennlp/commands/checklist.py (94%)
 rewrite allennlp/confidence_checks/task_checklists/__init__.py (100%)
```